tests/parsers/winreg_plugins/appcompatcache.py

```diff
@@ -37,11 +37,7 @@ def __init__(self, name):
     self.path_spec = fake_path_spec.FakePathSpec(location=name)
 
   def GetStat(self):
-    """Retrieves the stat object.
-
-    Returns:
-      dfvfs.VFSStat: stat object.
-    """
+    """Retrieves the stat object."""
```

Comment:
This docstring now kinda contradicts itself. Let's change the `return` to `return None` instead if you are so set on having a purposeful return section in the docstring.